src/models/User.ts

```diff
@@ -224,4 +294,58 @@ export class User extends MySqlModel {
       return this;
     }
   }
+
+  // Save the changes made to the User
+  async update(context: MyContext): Promise<User> {
```

Comment:
Not a big issue, but I noticed that additional tests weren't added for the User model, even with all these changes. So the coverage for User.ts is pretty low. But we can address that at a later time.

Reply:
Oh haha. I left an `it.only(` in place. I just removed it and made changes for the other issues you mentioned.

Reply:
Thanks Brian.